internal/service/service.go

```diff
@@ -0,0 +1,146 @@
+package service
+
+import (
+	"errors"
+	"github.com/VTerenya/employees/internal"
+	"github.com/VTerenya/employees/internal/repository"
+)
+
+type Handler interface {
+	CreatePosition(p internal.Position) error
+	CreateEmployee(e internal.Employee) error
+	GetPositions() []internal.Position
+	GetEmployees() []internal.Employee
+	GetPosition(id string) (internal.Position, error)
+	GetEmployee(id string) (internal.Employee, error)
+	DeletePosition(id string) error
+	DeleteEmployee(id string) error
+	UpdatePosition(p internal.Position) error
+	UpdateEmployee(e internal.Employee) error
+}
+
+type Service struct {
+	Handler
```

Comment:
here should be `repository` filed with corresponding interface as a type

Reply:
+